### DISCUSSION TITLE
Missing Mass Propers on 11 april 2024

### DISCUSSION THREAD

**Discussion by Michaelskapel** [DivinumOfficium/divinum-officium#3974] [Q&A] [answered]
For some languages (including Latin, English, Nederlands) some of the mass propers (Introitus, Lectio, Evangelium, Offertorium and Communio) do not show on 4-11-2024, while they do for most of the other languages. Can someone have a look and see what is going on?

**Comment by APMarcello3** [comment, marked as answer]
2fba3b8fe284532a37059e4c8a2a3d224687b04c
